src/modules/add-post/components/AddPost.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react'
+import axios from 'axios'
+import styles from '../styles/AddPost.module.css'
+
+export const AddPost = () => {
+    const handleSubmit = (e) => {
+        e.preventDefault()
+
+        const formData = new FormData(e.currentTarget)
+        const data = Object.fromEntries(formData)
+
+        console.log(data)
+
+        useEffect(() => {
+            axios
+                .post(
+                    'https://dpx3nob3b4anj5ujjhfudnsspu0ubhst.lambda-url.ap-southeast-2.on.aws/',
```

Comment:
lets change this to the localhost url. In future we change to the prod url from env file